config/jobs/kubernetes/sig-node/dra-ci.yaml

```diff
@@ -106,19 +98,19 @@ periodics:
             cpu: 2
             memory: 9Gi
 
-  # This job runs e2e_node.test with a focus on tests for the Dynamic Resource Allocation feature (currently beta)
   - name: ci-node-e2e-cgrpv1-crio-dra
-    cluster: k8s-infra-prow-build
+    cluster: eks-prow-build-cluster
```

Comment:
Shouldn't this remain unchanged?

https://github.com/kubernetes/test-infra/pull/34010#discussion_r1906045152